homeassistant/components/yolink/client.py

```diff
@@ -0,0 +1,308 @@
+"""Client of YoLink API."""
+
+import asyncio
+import logging
+from typing import Any, Dict, List
+import uuid
+
+from aiohttp.client import ClientResponse
+
+from homeassistant.components.yolink.device import YoLinkDevice
+from homeassistant.core import HomeAssistant, callback
+
+from .api import AuthenticationManager
+from .const import (
+    YOLINK_API_GATE,
+    YOLINK_API_MQTT_BROKER,
+    YOLINK_API_MQTT_BROKER_POER,
+    YoLinkAPIError,
+)
+from .model import BRDP
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class YoLinkHttpClient:
```

Comment:
This should be in a pypi package. You cannot have this in your component directly.

Reply:
Please close this PR. I submitted a new PR for yolink device integration #68682, In order to facilitate code review, i will restart my integration with a small PR

Reply:
Thank you. I'll revise it according to your suggestion